src/common/pubsub.ts

```diff
@@ -455,6 +459,12 @@ export const notifyPostYggdrasilIdSet = async (
     post,
   });
 
+export const notifyPostCollectionUpdated = async (
+  log: EventLogger,
+  collection: Pick<CollectionPost, 'id'>,
```

Comment:
I'm not 100% sure if this is alright, you are promising a `collection` but it's only a `collectionId` right?

Reply:
Our convention is to publish the ChangeObject from CDC to reduce DB calls down the line. Agree with Chris's point

Reply:
Yeah, this is leftover of mine when we did not use CDC, we will update the type. We actually already send change object just the type is wrong here.

Reply:
So this needs to be updated to `ChangeObject<CollectionPost>`

Reply:
Nevermind, I mixed `ChangeMessage` and `ChangeObject`